scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -21,6 +23,7 @@ VocabularyEditController.$inject = [
 ];
 
 interface IScope extends IScopeConfigController {
+    item?: Partial<IArticle>;
```

Comment:
better call it `fakeItem` and comment that it's `needed for editor3 directive(custom blocks)`